Gemfile.lock

```diff
@@ -271,11 +291,12 @@ DEPENDENCIES
   govuk-content-schema-test-helpers (~> 1.5)
   govuk-lint (~> 3.3)
   govuk_publishing_components!
+  govuk_schemas (~> 1.0)
   jasmine (~> 2.4.0)
   poltergeist (~> 1.16.0)
   rspec-rails (~> 3.6)
   uglifier (>= 1.3.0)
   webmock (~> 3.0.1)
 
 BUNDLED WITH
-   1.15.1
+   1.16.1
```

Comment:
Do we normally avoid changing this in commits?

Reply:
Good spot, will change back.